openlibrary/macros/FulltextSearchSuggestion.html

```diff
@@ -0,0 +1,35 @@
+$def with(query, results, page=1)
+
+$def render_snippet(query, doc):
+    $:macros.FulltextSuggestionSnippet(query, doc=doc)
+
+$if results and results.get('hits'):
+  $ hits = results['hits'].get('hits', [])[:4]
+  $ num_found = commify(results['hits'].get('total', 0))
+  $ ia_base_url = "https://archive.org"
+
+  $:macros.LoadingIndicator(_("Checking for Search Inside matches"))
+    <div id="fulltextSuggestions" class="fulltext-suggestions">
+      <div class="fulltext-suggestions__header">
+        <a class="fulltext-suggestions__icon" data-ol-link-track="SearchInsideSuggestion|SeeAllClick" href="/search/inside?$urlencode(dict(q=query))" title="$_('Search Inside Icon')">
+            <img src="/static/images/icons/icon_search-inside.svg"
+              width="30" height="30"
+              alt="$_('search inside icon')"/>
+        </a>
+        <h3 class="fulltext-suggestions__title">
+          <a data-ol-link-track="SearchInsideSuggestion|SeeAllClick"
```

Comment:
This `<a>` tag split across the internationalized text is a bit awkward -- @cdrini is this how we typically do it?

Also, is the `$:` necessary here?

Reply:
@rebecca-shoptaw do you have thoughts here?